src/main/resources/liquibase/202003271146-add-table-for-org-config.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
+
+    <changeSet id="202003271146-1" author="mstead">
+        <comment>Add table for org based configuration.</comment>
+        <createTable tableName="org_config">
+            <column name="org_id" type="VARCHAR(32)"/>
+            <column name="sync_enabled" type="BOOLEAN">
+                <constraints nullable="false" />
+            </column>
+            <column name="opt_in_type" type="VARCHAR(12)">
```

Comment:
Same thoughts as previous comment about `text` or `varchar(255)`.

Reply:
DONE - Went with varchar(255) since TEXT was causing issues with primary-keys. Decided to keep it consistent.